### ISSUE TITLE
makeVatConsole lacks methods of the SES VirtualConsole type

### ISSUE THREAD

**Issue by turadg** [Agoric/agoric-sdk#5453]
## What is the Problem Being Solved?

Some `console` method while coding in SDK are not available in swingset vats. The [error](https://github.com/Moddable-OpenSource/moddable/blob/fe78d8ac6e09f3e893359a93e237e260e2480e38/xs/sources/xsProxy.c#L177
) (e.g. for `console.trace`) is cryptic because it doesn't mention `console.trace` or even `console`, just the outer method.

Tests of these that run in Node will work, but then fail in XS tests which are harder to debug.

*Acceptance criteria*

the console type while coding in agoric-sdk matches the available console object during code execution

## Description of the Design

That could be by making Swingset's console support the [console API](https://developer.mozilla.org/en-US/docs/Web/API/console) or by modifying the typedef for the global console. I'd prefer to support the API as much as possible (if even with throwing "not implemented") than to create more special cases.

## Security Considerations

The [current approach ](https://github.com/Agoric/agoric-sdk/blob/9cb480c5f3de7b194f37f9932df56cdc126b1c51/packages/SwingSet/src/supervisors/supervisor-helper.js#L158-L170) puts console management in SDK when it would be safer to have lower in the environment.

## Test Plan

TBD

**Comment by dckc** [comment]
another way to phrase this would be as a bug:
 - `makeVatConsole` lacks `trace()` etc.; should match SES `VirtualConsole` type.

**Comment by turadg** [comment]
Fair enough. I was thinking it wouldn't merit a patch version bump but maybe so. I'll change the label.
